src/components/NewCommentForm.jsx

```diff
@@ -0,0 +1,27 @@
+import { HiUser } from "react-icons/hi2";
+import { postNewCommentByArticleId } from "../../utils/utils"
+import '../css/new_comment_form.css'
+
+export default function NewCommentForm({article_id, newComment, setNewComment, listOfComments, setListOfComments}) {
+    const handleNewComment = (e) => {
+            e.preventDefault();
+            postNewCommentByArticleId(article_id, newComment)
+                .then(({comment}) => {
+                    setNewComment({username: "", body: ""});
+                    setListOfComments((listOfComments) => {
+                        return [comment, ...listOfComments]
+                    })
+                })
+    }
+
+    return (
```

Comment:
Please disable the button or the whole form for the time of posting, so even the connection is slow, the user can't keep clicking and posting the same thing many times. 

Also a visual feedback would be nice to let the user know that the comment is being posted.